cnd/src/main.rs

```diff
@@ -23,21 +23,20 @@ use cnd::{
     db::Sqlite,
     file_lock::TryLockExclusive,
     http_api::route_factory,
-    jsonrpc,
-    lnd::LndConnectorParams,
-    load_swaps,
+    jsonrpc, load_swaps,
     network::Swarm,
     seed::RootSeed,
     swap_protocols::{
-        halight::InvoiceStates, Facade, Facade2, LedgerStates, SwapCommunicationStates,
-        SwapErrorStates,
+        halight::States, Facade, Facade2, LedgerStates, SwapCommunicationStates, SwapErrorStates,
     },
 };
 
```

Comment:
This line should not be here, the formatter didn't pick up the below use statements :)

Reply:
I have a fix locally and I'll push once I see how things are going.